makefiles/Kubernetes.mk

```diff
@@ -45,6 +45,7 @@ k8s-load-images:
 	./hack/k8s/minikube-load-images.sh
 
 # 5. Push to the container registry.
+#    Alternatively, you can `make k8s-load-images`.
 docker-push:
 	./hack/docker/push-local.sh
```

Comment:
This works much better for minikube than forwarding Docker registry.

Docker registry forward has a bunch of issues in WSL2<->Docker-for-Windows combo.